openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolderIngestion.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { uniqueId } from 'lodash';
+import { observer } from 'mobx-react';
+import React from 'react';
+
+const stepsData = [
+  {
+    step: 1,
+    title: 'Custom Airflow Installation',
```

Comment:
remove "Custom Airflow Installation" -> "Managed Airflow APIs"
OpenMetadata uses Airflow to run Ingestion Connectors. We developed Managed APIs to deploy ingestion connectors. Please refer to the guide here to use the OpenMetadata Airflow docker container or install the managed APIs in your Airflow installation.

Reply:
There should just be one Para, not multiple steps.
Show the error "Failed to find OpenMetadata - Managed Airflow APIs". below that add the text I wrote in first comment and link to the custom-airflow-installation

Reply:
Done